utils/RunONNXModel.py

```diff
@@ -868,6 +795,163 @@ def main():
                         "using atol={}, rtol={} ...".format(args.atol, args.rtol),
                     )
                     verify_outs(outs[i], ref_outs[i])
+        if outputname:
+            res = {outputname[i]: outs[i] for i in range(len(outs))}
+            return res
+        else:
+            return outs
+
+
+"""
+in onnxruntime:
+class onnxruntime.InferenceSession(path_or_bytes: str | bytes | os.PathLike, sess_options: onnxruntime.SessionOptions | None = None, providers: Sequence[str | tuple[str, dict[Any, Any]]] | None = None, provider_options: Sequence[dict[Any, Any]] | None = None, **kwargs)[source]
+
+In onnxmlir, session_options and provider will be merged into kwargs, and
+ignored. onnxruntime.SessionOptions may contain some useful info,
+but onnxruntime package is needed to interpret it. Therefore, it is ignored now.
+Another argument, 'options' is added for onnxmlir to specify options for RunONNXModel.py
+"""
+
+
+def inferenceSession(model_name, **kwargs):
+    global args
```

Comment:
Can we move this code into the `__init__` of `InferenceSession` class and remove `inferenceSession` method? The reason is it is a bit confusing about `InferenceSession` class and `inferenceSession` method. Other parts looks good to me.

Reply:
Fixed. Thanks for the suggestion.